scripts/style.sh

```diff
@@ -92,29 +92,7 @@ function join() {
 }
 
 clang_options=(-style=file)
-
-# Rules to disable in swiftformat:
-swift_disable=(
-  # sortedImports is broken, sorting into the middle of the copyright notice.
-  sortedImports
-
-  # Too many of our swift files have simplistic examples. While technically
-  # it's correct to remove the unused argument labels, it makes our examples
-  # look wrong.
-  unusedArguments
-
-  # We prefer trailing braces.
-  wrapMultilineStatementBraces
-)
-
-swift_options=(
-  # Mimic Objective-C style.
-  --indent 2
-  --maxwidth 100
-  --wrapparameters afterfirst
-
-  --disable $(join , "${swift_disable[@]}")
-)
+swift_options=()
```

Comment:
Please add a comment about how options are set.

Reply:
Good call. Added a comment in [`d30b847` (#13423)](https://github.com/firebase/firebase-ios-sdk/pull/13423/commits/d30b8473782cfb85521fdda1ca52057c03724e57).